packages/behandling-fp/src/BehandlingForeldrepengerIndex.tsx

```diff
@@ -24,6 +24,7 @@ const foreldrepengerData = [
   { key: FpBehandlingApiKeys.INNTEKT_ARBEID_YTELSE },
   { key: FpBehandlingApiKeys.BEREGNINGRESULTAT_FORELDREPENGER },
   { key: FpBehandlingApiKeys.BEREGNINGSGRUNNLAG },
+  { key: FpBehandlingApiKeys.FERIEPENGEGRUNNLAG },
```

Comment:
Denne burde vel heller ligga i getEndepunkter i TilkjentYtelseProsessStegPanelDef.tsx? Skal kun ligga i denne lista om ein treng data for å utleda visning av panel. Ligg den her blir det alltid henta, om den ligg i *PanelDef blir den kun henta når ein vises aktuelt panel